package-lock.json

```diff
@@ -7573,8 +7573,7 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true,
-          "optional": true
```

Comment:
Were these changes intentional or just happened when you re-generated the lock file? We might want to just leave them out if we're not consciously changing dependencies. Same thing happened to me due to running a different version of npm, fixed it by changing to v6.11.3 (node v10.17.0)

Reply:
this is the lock after running `npm run build | Builds the production assets.` i'd assume it's ok have this committed

Reply:
after looking into `dev true` & `optional true`:
```If true then this dependency is either a development dependency ONLY of the top level module or a transitive dependency of one. This is false for dependencies that are both a development dependency of the top level and a transitive dependency of a non-development dependency of the top level.```

Reply:
Yep, just wanted to reduce upstream diff if possible but agreed I think it's fine.